### PR TITLE
Redux components

### PR DESCRIPTION
**Pull request by JuanLPalacios**
use Redux store to display books. and add events to components to make them dispatch actions.

### CODE REVIEW

File: src/Categories.js [hunk 1]
```diff
@@ -1,3 +1,7 @@
+import { useDispatch } from 'react-redux';
+import { checkCargory } from './redux/categories/categories';
+
 export default function Categories() {
-  return <><button type="button">Check status</button></>;
+  const dispatch = useDispatch();
+  return <><button type="button" onClick={dispatch(checkCargory())}>Check status</button></>;
```
Comment (er-contreras) [inline src/Categories.js]: - [x] The button "Check Status" should show the message "Under Construction"

**Review by er-contreras — CHANGES_REQUESTED** [review]
Hi @JuanLPalacios, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

almost there

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- To keep your Books and Categories components in the same folder to keep everything together and more in order. 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/Categories.js (hunk 1 above)
Comment (JuanLPalacios) [reply src/Categories.js]: check the documentation only dispatching the action is required

**Review by BrianSammit — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

Hello @JuanLPalacios, 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

TO HIGHLIGHT:

- Your project is professional 
- Your repo is professional 
- Well structured files 


Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 

well-1

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**